src/main/java/teamrtg/rtg/core/RTG.java

```diff
@@ -38,11 +44,12 @@ public void preInit(FMLPreInitializationEvent event) {
 
         configPath = event.getModConfigurationDirectory() + "/RTG/";
         Mods.syncAllConfigs();
+        
+        // This must come before the event manager is registered.
+        this.registerStructures();
```

Comment:
>I guess I was just trying to keep the preInit a bit cleaner, but happy to move them back.

Doesn't really matter too much, but if you're going to go through the effort to put that code into a separate method to keep the main class cleaner, then you may as well put it into static methods in the related classes, and call the method there during pre-init, that way the main class only has the method calls. :)